library/scripts/common_bd.tcl

```diff
@@ -0,0 +1,96 @@
+################################################################################
+################################################################################
+
```

Comment:
License header should be added.

Reply:
Our Tcl scripts don't have licence headers... Do we start adding, or is this a confusion?

Reply:
I think it is a good idea to have license headers for all files. Some of our tcl file have them.